css/font.css

```diff
@@ -0,0 +1,14 @@
+﻿@font-face {
+  font-family: 'GothaPro';
+  src: url('../fonts/GothaProReg.eot');
+  src: url('../fonts/GothaProReg.woff') format('woff'), url('../fonts/GothaProReg.woff2') format('woff2'),url('../fonts/GothaProReg.ttf') format('truetype'), url('../fonts/GothaProReg.svg') format('svg');
+  font-weight: normal;
+  font-style: normal;
+}
+@font-face {
+  font-family: 'GothaPro';
+  src: url('../fonts/GothaProBol.eot');
+  src: url('../fonts/GothaProBol.woff') format('woff'), url('../fonts/GothaProBol.woff2') format('woff2'), url('../fonts/GothaProBol.ttf') format('truetype'), url('../fonts/GothaProBol.svg') format('svg');
+  font-weight: bold;
+  font-style: normal;
+  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/axion_ann/pullRequest?prid=5594413)